src/components/SingleBlog/index.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import "./index.scss";
+import { Container, Col, Row } from "react-bootstrap";
+import { useParams } from "react-router-dom";
+import { ReactComponent as FacebookIcon } from "../../images/icons/facebookIcon.svg";
+import { ReactComponent as TwitterIcon } from "../../images/icons/twitterIcon.svg";
+import { ReactComponent as LinkedInIcon } from "../../images/icons/linkedInIcon.svg";
+import { ReactComponent as InstagramIcon } from "../../images/icons/instagramIcon.svg";
+
+const RELATED_POSTS_TO_SHOW = 4;
+
+export default function SingleBlog(props) {
+  const params = useParams();
+  const [blogPost, setBlogPosts] = useState();
+  const allBlogPosts = async () => {
+    const data = await fetch(
+      `https://public-api.wordpress.com/wp/v2/sites/safeplace102505649.wordpress.com/posts/${params.id}`
+    );
+    const posts = await data.json();
+    setBlogPosts(posts);
+  };
+  useEffect(() => {
+    allBlogPosts();
+  });
+
+  if (!blogPost) {
+    return <h1>loading</h1>;
+  } else {
+    return (
+      <Container>
+        <Row className="blogContent">
+          <Col xs={12} md={8}>
+            <a href="./">
+              <h5 className="backToBlogBtn">
+                <span>&#8592;</span>Back to blog
+              </h5>
+            </a>
+            <div
+              className="article"
+              dangerouslySetInnerHTML={{
+                __html: blogPost.content.rendered,
+              }}
+            ></div>
+            <h4>Share on social media:</h4>
+            <div className="socialMediaIcons">
+              <FacebookIcon />
+              <TwitterIcon />
+              <LinkedInIcon />
+              <InstagramIcon />
+            </div>
+          </Col>
+          <Col md={4} className="relatedPosts">
+            <h5>Related articles:</h5>
+            <div className="firstRelatedPost">
```

Comment:
class name is not correct here it's says firstRelatedPost but it's mapping all posts. 
and you can check maybe if there extra div

Reply:
class names changed in the related post container